### PR TITLE
Issue #7236 Fixed

### PR DESCRIPTION
**Pull request by suraj-webkul**
#BUG 
#7236 
>Fixed UI Issue

### CODE REVIEW

**Comment by Sanjaybhattwebkul** [comment]
@suraj-webkul 
**Not fixed,** Please find the screenshot [Here](https://prnt.sc/Ipy43gS8AoPP)

**Comment by suraj-webkul** [comment]
> @suraj-webkul **Not fixed,** Please find the screenshot [Here](https://prnt.sc/Ipy43gS8AoPP)

Added the publishable assets please check again

**Comment by Sanjaybhattwebkul** [comment]
@suraj-webkul 
**Still not fixed**, if there is no subcategory then there is a UI issue. Please find the screenshot [Here](https://prnt.sc/ZDnOmwTHlQ9W)

**Comment by Sanjaybhattwebkul** [comment]
> @suraj-webkul **Still not fixed**, if there is no subcategory then there is a UI issue. Please find the screenshot [Here](https://prnt.sc/ZDnOmwTHlQ9W)

**Comment by suraj-webkul** [comment]
@Sanjaybhattwebkul 
Screenshot from 2023-01-20 16-17-19

This is working on my system

**Review by Sanjaybhattwebkul — DISMISSED** [review]

**Comment by Sanjaybhattwebkul** [comment]
@suraj-webkul 

Please resolve the conflicts.

File: packages/Webkul/Product/src/Repositories/ProductRepository.php [hunk 1]
```diff
@@ -73,11 +73,13 @@ public function update(array $data, $id, $attribute = 'id')
 
         $product = $product->getTypeInstance()->update($data, $id, $attribute);
 
+        $updatedProduct = $this->find($id);
```
Comment (devansh-webkul) [inline packages/Webkul/Product/src/Repositories/ProductRepository.php:76]: No need of find method again.

File: packages/Webkul/Shop/src/DataGrids/GuestOrderDataGrid.php [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+     */
+    protected $index = 'id';
+
+    /**
+     * Sort order.
+     *
+     * @var string
+     */
+    protected $sortOrder = 'desc';
+
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/src/DataGrids/GuestOrderDataGrid.php:23]: Extra new lines.

**Comment by Sanjaybhattwebkul** [comment]
@suraj-webkul 
Getting **duplicate commits**. please **resolve**

**Comment by Sanjaybhattwebkul** [comment]
@suraj-webkul 
This is **working fine.**

**Review by Sanjaybhattwebkul — APPROVED** [review]

File: packages/Webkul/Product/src/Repositories/ProductRepository.php (hunk 1 above)
Comment (suraj-webkul) [reply packages/Webkul/Product/src/Repositories/ProductRepository.php]: Done

File: packages/Webkul/Velocity/src/Resources/assets/sass/app.scss [hunk 3]
```diff
@@ -136,6 +136,15 @@ label {
     text-decoration: none;
 }
 
+/*
+    default hover theme
+*/
+.default-hover-text{
```
Comment (akathuria289) [inline packages/Webkul/Velocity/src/Resources/assets/sass/app.scss:142]: Add a space before the bracket @suraj-webkul

File: packages/Webkul/Velocity/src/Resources/assets/sass/components/home.scss [hunk 4]
```diff
@@ -386,7 +388,7 @@ header #search-form > *:focus {
     }
 
     .popular-category-wrapper:nth-last-child(2) {
-        padding-right: 0px;
+        // padding-right: 0px;
```
Comment (akathuria289) [inline packages/Webkul/Velocity/src/Resources/assets/sass/components/home.scss:391]: Remove the commented code @suraj-webkul

Comment (suraj-webkul) [reply packages/Webkul/Velocity/src/Resources/assets/sass/components/home.scss]: Done

File: packages/Webkul/Velocity/src/Resources/assets/sass/app.scss (hunk 3 above)
Comment (suraj-webkul) [reply packages/Webkul/Velocity/src/Resources/assets/sass/app.scss]: done

**Pull request closed without merging**